set.mm

```diff
@@ -643200,6 +643206,781 @@ base set if and only if the neighborhoods (convergents) of every point
+  $}
+
+  ${
+    simpgntrivd.1 $e |- B = ( Base ` G ) $.
+    simpgntrivd.2 $e |- .0. = ( 0g ` G ) $.
+    simpgntrivd.3 $e |- ( ph -> G e. SimpGrp ) $.
+    $( Simple groups are nontrivial.  (Contributed by Rohan Ridenour,
+       3-Aug-2023.) $)
+    simpgntrivd $p |- ( ph -> -. B = { .0. } ) $=
+      ( csn wceq csimpg wcel adantr cgrp simpggrpd simpr trivnsimpgd pm2.65da
+      wa ) ABDHIZCJKZATSGLASRBCDEFACMKSACGNLASOPQ $.
+  $}
+
+  ${
+    $d x .0. $.  $d x B $.
+    simpgnideld.1 $e |- B = ( Base ` G ) $.
+    simpgnideld.2 $e |- .0. = ( 0g ` G ) $.
+    simpgnideld.3 $e |- ( ph -> G e. SimpGrp ) $.
+    $( A simple group contains a nonidentity element.  (Contributed by Rohan
+       Ridenour, 3-Aug-2023.) $)
+    simpgnideld $p |- ( ph -> E. x e. B -. x = .0. ) $=
+      ( cv wceq wral wn wrex csn simpgntrivd c0 wne wb cgrp wcel cmnd simpggrpd
+      grpmnd mndidcl 3syl ne0d eqsn syl mtbid rexnal sylibr ) ABIEJZBCKZLULLBCM
+      ACENJZUMACDEFGHOACPQUNUMRACEADSTDUATECTADHUBDUCCDEFGUDUEUFBCEUGUHUIULBCUJ
+      UK $.
+  $}
+
+  ${
+    simpgnsgd.1 $e |- B = ( Base ` G ) $.
+    simpgnsgd.2 $e |- .0. = ( 0g ` G ) $.
+    simpgnsgd.3 $e |- ( ph -> G e. SimpGrp ) $.
+    $( The only normal subgroups of a simple group are the group itself and the
+       trivial group.  (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
+    simpgnsgd $p |- ( ph -> ( NrmSGrp ` G ) = { { .0. } , B } ) $=
+      ( cnsg cfv csn c2o cfn wcel cen wbr a1i syl syl2anc cvv cbs cpr 2onn nnfi
+      com simpg2nsg enfii simpggrpd 0idnsgd snex wceq fvex syl6eqel simpgntrivd
+      csimpg neqcomd enpr2d ensymd entr rr-php2d ) ACHIZDJZBUAZAKLMZUTKNOZUTLMA
+      KUDMZVCVEAUBPKUCQACUNMVDGCUEQZUTKUFRABCDEFACGUGUHAVDKVBNOUTVBNOVFAVBKAVAB
+      SSVASMADUIPABCTIZSBVGUJAEPCTUKULABVAABCDEFGUMUOUPUQUTKVBURRUS $.
+  $}
+
+  ${
+    simpgnsg2d.1 $e |- B = ( Base ` G ) $.
+    simpgnsg2d.2 $e |- .0. = ( 0g ` G ) $.
+    simpgnsg2d.3 $e |- ( ph -> G e. SimpGrp ) $.
+    simpgnsg2d.4 $e |- ( ph -> A e. ( NrmSGrp ` G ) ) $.
+    $( A normal subgroup of a simple group is either the whole group or the
+       trivial subgroup.  (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
+    simpgnsg2d $p |- ( ph -> ( A = { .0. } \/ A = B ) ) $=
+      ( csn cpr wcel wceq wo cnsg cfv simpgnsgd eleqtrd elpri syl ) ABEJZCKZLBU
+      AMBCMNABDOPUBIACDEFGHQRBUACST $.
+  $}
+
+  ${
+    $d ph x $.  $d x .0. $.  $d x B $.  $d x G $.
+    2nsgsimpgd.1 $e |- B = ( Base ` G ) $.
+    2nsgsimpgd.2 $e |- .0. = ( 0g ` G ) $.
+    2nsgsimpgd.3 $e |- ( ph -> G e. Grp ) $.
+    2nsgsimpgd.4 $e |- ( ph -> -. { .0. } = B ) $.
+    2nsgsimpgd.5 $e |- ( ( ph /\ x e. ( NrmSGrp ` G ) ) ->
+      ( x = { .0. } \/ x = B ) ) $.
+    $( If any normal subgroup of a nontrivial group is either the trivial
+       subgroup or the whole group, the group is simple.  (Contributed by Rohan
+       Ridenour, 3-Aug-2023.) $)
+    2nsgsimpgd $p |- ( ph -> G e. SimpGrp ) $=
```

Comment:
It would be nice to combine ~simpgnsgd and ~ 2nsgsimpgd to, e.g., `` ( ph -> ( G e. SimpGrp ) <-> ( NrmSGrp ` G ) = { { .0. } , B } ) )`` to prove the statement given in the definition: "A simple group is a group with exactly two normal subgroups." or "A nontrivial group is simple iff it has exactly two normal subgroups".  This theorem could/should be references in the comment of the definition.

Reply:
Added as ~ simpgnsgbid.